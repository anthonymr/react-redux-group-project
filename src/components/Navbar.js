import styles from '../styles/Navbar.module.css';
import planet from '../assets/planet.png';
import Route from './Route';

function Navbar() {
  return (
    <header className={styles.header}>
      <div className={styles.logoContainer}>
        <img src={planet} alt="Space Travelers&apos; Hub" />
        <span>Space Travelers&apos; Hub</span>
      </div>
      <nav className={styles.nav}>
        <Route />
      </nav>
    </header>
  );
}
export default Navbar;
