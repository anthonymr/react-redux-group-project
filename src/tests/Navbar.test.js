import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import Navbar from '../components/Navbar';

it('Footer renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Navbar />
      </MemoryRouter>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
