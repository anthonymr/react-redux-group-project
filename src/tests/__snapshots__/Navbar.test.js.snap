// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<header
  className="header"
>
  <div
    className="logoContainer"
  >
    <img
      alt="Space Travelers' Hub"
      src="planet.png"
    />
    <span>
      Space Travelers' Hub
    </span>
  </div>
  <nav
    className="nav"
  >
    <ul>
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li>
        <a
          className=""
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
      </li>
      <li>
        |
      </li>
      <li>
        <a
          className=""
          href="/myprofile"
          onClick={[Function]}
        >
          My profile
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
